research/src/pages/design-systems.mdx

```diff
@@ -0,0 +1,16 @@
+---
+menu: Overview
+name: Design Systems
+path: /analysis/design-systems
+---
+
+import DesignSystemsList from '../components/design-systems-list'
+
+# List of analyzed design systems
+
+This table shows names and links to design systems which are currently being used during the research phase.
```

Comment:
See comment on Una's suggestion.

Reply:
Ping on this :) should be good to go after

Reply:
@una Finally done.